resources/options_lookup.tsv

```diff
@@ -477,14 +477,14 @@ Insulation Ceiling	R-30	ResidentialConstructionsUnfinishedAttic	ceiling_r=30	cei
+Insulation Roof	"Finished, Uninsulated"	ResidentialConstructionsUnfinishedAttic	ceiling_r=0	ceiling_install_grade=1	ceiling_ins_thick_in=0	ceiling_framing_factor=0.07	ceiling_joist_height_in=3.5	ceiling_drywall_thick_in=0.5	roof_cavity_r=0	roof_install_grade=1	roof_cavity_ins_thick_in=0	roof_framing_factor=0.07	roof_framing_thick_in=7.25	roof_osb_thick_in=0.75	roof_rigid_r=0																																
+Insulation Roof	"Finished, R-7"	ResidentialConstructionsUnfinishedAttic	ceiling_r=7	ceiling_install_grade=1	ceiling_ins_thick_in=0	ceiling_framing_factor=0.07	ceiling_joist_height_in=3.5	ceiling_drywall_thick_in=0.5	roof_cavity_r=7	roof_install_grade=1	roof_cavity_ins_thick_in=2.95	roof_framing_factor=0.07	roof_framing_thick_in=7.25	roof_osb_thick_in=0.75	roof_rigid_r=0																																
+Insulation Roof	"Finished, R-13"	ResidentialConstructionsUnfinishedAttic	ceiling_r=13	ceiling_install_grade=1	ceiling_ins_thick_in=0	ceiling_framing_factor=0.07	ceiling_joist_height_in=3.5	ceiling_drywall_thick_in=0.5	roof_cavity_r=13	roof_install_grade=1	roof_cavity_ins_thick_in=5.47	roof_framing_factor=0.07	roof_framing_thick_in=7.25	roof_osb_thick_in=0.75	roof_rigid_r=0																																
+Insulation Roof	"Finished, R-19"	ResidentialConstructionsUnfinishedAttic	ceiling_r=19	ceiling_install_grade=1	ceiling_ins_thick_in=0	ceiling_framing_factor=0.07	ceiling_joist_height_in=3.5	ceiling_drywall_thick_in=0.5	roof_cavity_r=19	roof_install_grade=1	roof_cavity_ins_thick_in=7.41	roof_framing_factor=0.07	roof_framing_thick_in=7.25	roof_osb_thick_in=0.75	roof_rigid_r=0																																
+Insulation Roof	"Finished, R-30"	ResidentialConstructionsUnfinishedAttic	ceiling_r=30	ceiling_install_grade=1	ceiling_ins_thick_in=0	ceiling_framing_factor=0.07	ceiling_joist_height_in=3.5	ceiling_drywall_thick_in=0.5	roof_cavity_r=30	roof_install_grade=1	roof_cavity_ins_thick_in=11.23	roof_framing_factor=0.07	roof_framing_thick_in=7.25	roof_osb_thick_in=0.75	roof_rigid_r=0																																
+Insulation Roof	"Finished, R-38"	ResidentialConstructionsUnfinishedAttic	ceiling_r=38	ceiling_install_grade=1	ceiling_ins_thick_in=0	ceiling_framing_factor=0.07	ceiling_joist_height_in=3.5	ceiling_drywall_thick_in=0.5	roof_cavity_r=38	roof_install_grade=1	roof_cavity_ins_thick_in=14.01	roof_framing_factor=0.07	roof_framing_thick_in=7.25	roof_osb_thick_in=0.75	roof_rigid_r=0																																
+Insulation Roof	"Finished, R-49"	ResidentialConstructionsUnfinishedAttic	ceiling_r=49	ceiling_install_grade=1	ceiling_ins_thick_in=0	ceiling_framing_factor=0.07	ceiling_joist_height_in=3.5	ceiling_drywall_thick_in=0.5	roof_cavity_r=49	roof_install_grade=1	roof_cavity_ins_thick_in=17.84	roof_framing_factor=0.07	roof_framing_thick_in=7.25	roof_osb_thick_in=0.75	roof_rigid_r=0																																

```

Comment:
@aspeake Shouldn't `ceiling_r` be zero for all of these? Otherwise you're modeling both ceiling and roof insulation.

Reply:
Yep, thanks